### ISSUE TITLE
handler.js defined in package.json but it's not in the project

### ISSUE THREAD

**Issue by vincentclaes** [AnomalyInnovations/serverless-stack-demo-api#21]
there is this line  in the package.json

`"main": "handler.js",
`
but the handler file is not present in the project. is this a mistake? does this run?

thanks.

vincent

**Comment by jayair** [comment]
@vincentclaes Good catch. We don't use this, so it doesn't affect anything. But we don't need it there either.
